### ISSUE TITLE
[maps/comedy_club] Interior Upgrade

### ISSUE THREAD

**Issue by AldoLEGACY** [InZidiuZ/legacy-assets-issue-tracker#376]
**Name:**

ALDO#4439

**Summary:**

I present a suggestion to change the interior of a comedy club. I am the leader of a criminal organization operating on the Polish Legacy and I plan to buy a club, but I noticed that clubs and restaurants do not live up to expectations - they are poor. At the Bahamas Mamas club, you can't even put a person behind a bar without teleporting. I am providing a link to the club we would like to buy if it is replaced. The present interior of the comedy club is simply unplayable.

**Link:**
https://www.gta5-mods.com/maps/split-sides-west-mlo-fivem-ragemp
https://www.youtube.com/watch?v=IzIOpxpQCjk

**Reason:**

I believe that not only me, but also many players, would be very pleased with this new interior. It's bigger, but it's not huge. It has a much better atmosphere, you can organize roasts, concerts, parties, etc. in it. This interior has much greater potential and you can organize many more RP actions in it. In addition, it is a great place for dark interests for the mafia or other criminal organizations.

**Comment by MsBBBones** [comment]
Hi! My name is BB Bones   (BertBones#2590) and I currently own the Split Sides Comedy Club in the Legacy EU server! I was reading over your post, and I think an upgrade to the interior would be a fantastic idea that could benefit not only the city’s economy, but the nightlife altogether. The comedy club is located at a very strategic and convenient spot in the city, and with it receiving a GPS icon recently, it’s noticeable even more so now. The club has received quite the popularity strike since its opening and has been often referred to as “One of the funnest places to hang out in”. The events that have been hosted there have been nothing short of fun for both the public and the club’s staff team working the shifts! There is, however, many limitations that could be lifted for the betterment of the club & business as a whole. The other day, for example, an event was hosted at the comedy club in the form of a roast to one of our most cherished community members. The event was full to more than capacity and there came a time where the doors had to be closed and citizens were not allowed to go inside due to it being extremely full (despite there being 10+ people outside eager to get in). An upgrade to the interior of this club would mean bigger events that would benefit the city’s day and nightlife. 
   It would mean the promotion of more jobs for new & old players alike in the private sector, because although we are allowed unlimited bartenders, the size of the club only limits us to two per event before it gets too crowded behind the bar, and the size of the club in general doesn’t warrant much mobility throughout. This remodeling and upgrade would allow for the possibility of stationary bartenders placed behind the bar, as well as constantly shifting ones that could bring food and drinks to the customers in the comfort of their own table. 
 It could mean future collaborations with local businesses to further promote and boost them, providing social and financial stability towards each other, which could warrant more constant and active businesses/events, which would inevitably provide more roleplay options for members who aren’t interested in crime rp and simply want to maintain an active social life. It could, in all honesty, mean the potential for a club that’s open at all times of the day, with constantly shifting staff members. The possibilities are truly endless! 
 In conclusion, wholeheartedly back this suggestion up and strongly believe this remodeling/upgrade is an investment towards the betterment & enjoyment of the city as a whole; providing a more active nightlife, the promotion of more lucrative and active jobs for new & existing players alike in the private sector, and the constant flow of money to and from the community. I look forward to hearing more updates on this post, and I thank you for making such a wonderful suggestion!

- BB

**Comment by punanipress** [comment]
This is gorgeous!~ +1

**Comment by AldoLEGACY** [comment]
> Hi! My name is BB Bones (BertBones#2590) and I currently own the Split Sides Comedy Club in the Legacy EU server! I was reading over your post, and I think an upgrade to the interior would be a fantastic idea that could benefit not only the city’s economy, but the nightlife altogether. The comedy club is located at a very strategic and convenient spot in the city, and with it receiving a GPS icon recently, it’s noticeable even more so now. The club has received quite the popularity strike since its opening and has been often referred to as “One of the funnest places to hang out in”. The events that have been hosted there have been nothing short of fun for both the public and the club’s staff team working the shifts! There is, however, many limitations that could be lifted for the betterment of the club & business as a whole. The other day, for example, an event was hosted at the comedy club in the form of a roast to one of our most cherished community members. The event was full to more than capacity and there came a time where the doors had to be closed and citizens were not allowed to go inside due to it being extremely full (despite there being 10+ people outside eager to get in). An upgrade to the interior of this club would mean bigger events that would benefit the city’s day and nightlife.
> It would mean the promotion of more jobs for new & old players alike in the private sector, because although we are allowed unlimited bartenders, the size of the club only limits us to two per event before it gets too crowded behind the bar, and the size of the club in general doesn’t warrant much mobility throughout. This remodeling and upgrade would allow for the possibility of stationary bartenders placed behind the bar, as well as constantly shifting ones that could bring food and drinks to the customers in the comfort of their own table.
> It could mean future collaborations with local businesses to further promote and boost them, providing social and financial stability towards each other, which could warrant more constant and active businesses/events, which would inevitably provide more roleplay options for members who aren’t interested in crime rp and simply want to maintain an active social life. It could, in all honesty, mean the potential for a club that’s open at all times of the day, with constantly shifting staff members. The possibilities are truly endless!
> In conclusion, wholeheartedly back this suggestion up and strongly believe this remodeling/upgrade is an investment towards the betterment & enjoyment of the city as a whole; providing a more active nightlife, the promotion of more lucrative and active jobs for new & existing players alike in the private sector, and the constant flow of money to and from the community. I look forward to hearing more updates on this post, and I thank you for making such a wonderful suggestion!
> 
> * BB

I agree with everything you wrote. With such an interior, we can create the number one place for everyone to spend time. At the moment, the Vanilla Unicorn strip club dominates Legacy. I'm not saying it's bad, it's also great, but there is no place for everyone to hang out. I would call this place "a good place for anyone who wants to have a good time".

**Comment by JacobTrixx** [comment]
UPVOTE FOR BBS COMEDY CLUB

**Comment by Nuziatella** [comment]
This will be looked into.

**Comment by InZidiuZ** [comment]
This has now been added.
